app/src/main/java/framgia/com/ichat/screen/privateroom/PrivateRoomAdapter.java

```diff
@@ -0,0 +1,77 @@
+package framgia.com.ichat.screen.privateroom;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import framgia.com.ichat.GlideApp;
+import framgia.com.ichat.R;
+import framgia.com.ichat.data.model.Message;
+import framgia.com.ichat.data.model.PrivateRoom;
+
+public class PrivateRoomAdapter extends RecyclerView.Adapter<PrivateRoomAdapter.ViewHolder> {
+    private Context mContext;
+    private List<PrivateRoom> mPrivateRooms;
+    private LayoutInflater mLayoutInflater;
+
+    public PrivateRoomAdapter(Context context, List<PrivateRoom> privateRooms, LayoutInflater layoutInflater) {
```

Comment:
sao lại đưa LayoutInflater vào đây vậy e?
```suggestion
    public PrivateRoomAdapter(Context context, List<PrivateRoom> privateRooms) {
               mLayoutInflater = LayoutInflater.from(mContext);
```